src/gui/src/search.cpp

```diff
@@ -363,16 +354,12 @@ void Search::updateBlockages(odb::dbBlock* block)
 
   data.blockages_.clear();
 
-  std::vector<BoxValue<odb::dbBlockage*>> blockages;
+  std::vector<DBoxValue<odb::dbBlockage*>> blockages;
   for (odb::dbBlockage* blockage : block->getBlockages()) {
-    odb::dbBox* bbox = blockage->getBBox();
-    const Point ll(bbox->xMin(), bbox->yMin());
-    const Point ur(bbox->xMax(), bbox->yMax());
-    const Box box(ll, ur);
-    blockages.emplace_back(box, blockage);
+    blockages.emplace_back(blockage->getBBox(), blockage);
```

Comment:
Why do we need to store both?  Couldn't we just have the index getter work directly on the blockage?  Other types may be similar.

Reply:
I think we can, I got confused by some slowdowns in the rendering (I just sent you an email about it), which I thought might have been related to this, but now I don't. I'll try switching the dbBox stuff over and if there isn't an measurable rendering time impact, I'll push it up.